src/GridView.js

```diff
@@ -1,13 +1,6 @@
-import React, {
-  Component,
-  PropTypes,
-} from 'react';
-import {
-  Dimensions,
-  ListView,
-  StyleSheet,
-  View,
-} from 'react-native';
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
```

Comment:
Pull in the default export of the `prop-types` package as `PropTypes`, instead of getting `PropTypes` out of the `react` package.